main/models.py

```diff
@@ -130,3 +134,43 @@ def to_dict(self):
             "type": self.type,
             "expense_date": self.expense_date,
         }
+
+class Payroll(models.Model):
+    profile = models.OneToOneField(Profile, on_delete=models.CASCADE, related_name="payroll", null=True)
+    basic_salary = models.DecimalField(max_digits=10, decimal_places=2)
+    hra = models.DecimalField(max_digits=10, decimal_places=2, default=0.0)
+    special_allowance = models.DecimalField(max_digits=10, decimal_places=2, default=0.0)
+    tax_deduction = models.DecimalField(max_digits=10, decimal_places=2, default=0.0)
+    reimbursement_amount = models.DecimalField(max_digits=10, decimal_places=2, default=0.0)
+    final_salary = models.DecimalField(max_digits=10, decimal_places=2, default=0.0)
```

Comment:
if someday someone edits the hra for eg, and forgets to change this then ? Why are there dependencies in the same table ?